
hiPSCs lines
The above mentioned microglial and astrocyte functioning assays will be performed for both control and KdVS derived hiPSCs. The hiPSCs lines were chosen as described in (Verboven et al., 2024). 3 control lines (C1, C2, C3) along with 4 KdVS lines (KdVS1,2,3,4). All hiPSCs were derived from fibroblasts from either healthy controls or a full spectrum of KdVS variations (Koolen et al., 2016) to ensure the entire spectrum of symptoms due to KdVS are included. These hiPSCs lines were used for microglial and astrocyte functioning assays, however for the MEA sequencing method, only one control line and one KdVS line was selected as this will be a quad culture and too many lines would result in excessive control and KdVS groups for each cell type. 

Inhibitory neurons
To ensure that the hiPSCs lines differentiate into inhibitory neurons, the the modified protocol specified in (Weidling et al., 2025) was used. This is done by differentiating the hiPSCs line first into ventral forebrain neural progenitors and eventually into cortical inhibitory neurons. 

Excitatory neurons
All lines had the transgene neurogenin2 integrated into the genome such that all the lines of hiPSCs are generated into rtTA/Ngn2-positive hiPSCs as described previously (Linda et al., 2022). These were then differentiated by doxycycline-induced overexpression of Neurogenin-2 (Ngn2) into cortical exhitatory neurons (Frega et al., 2017). 

Astrocytes
(Schuurmans et al., 2024)

Microglia
The method mentioned in (Mordelt et al., 2025) was used

MEA recordings
Wells
Novel method
What is being measured


Different combinations of quad cultures
To find which neuronal and glial cell is most affected be KANSL1 in terms of networks firing, multiple groups of all possible permutation of control and KdVS cells quad culture were created. This consists of 16 total groups ensuring there is every combination of cell type, both control and KdVSs, to find the 

RNA sequencing 
When

MEA-seq correlation analysis
Correlation and additional analysis

![[Screenshot 2025-11-30 at 19.22.56.png]]


A.H.A. Verboven, Puvogel, S., N. Kolsters, Latour, B., Linda, K., E.I. Lewerissa, R. Mijdam, Dyke, E., D. Duineveld, M. Zoodsma, Claassen, W. J. J., A. Oudakker, C. Schoenmaker, D.A. Koolen, B.B.A. de Vries, Hoen, P. A. C. t, & N. Nadif Kasri. (2024). Integrative transcriptomics and electrophysiological profiling of hiPSC-derived neurons identifies novel druggable pathways in Koolen-de Vries Syndrome. _BioRxiv (Cold Spring Harbor Laboratory)_. https://doi.org/10.1101/2024.08.29.610281

Koolen, D. A., Pfundt, R., Linda, K., Beunders, G., Veenstra-Knol, H. E., Conta, J. H., Fortuna, A. M., Gillessen-Kaesbach, G., Dugan, S., Halbach, S., Abdul-Rahman, O. A., Winesett, H. M., Chung, W. K., Dalton, M., Dimova, P. S., Mattina, T., Prescott, K., Zhang, H. Z., Saal, H. M., & Hehir-Kwa, J. Y. (2015). The Koolen-de Vries syndrome: a phenotypic comparison of patients with a 17q21.31 microdeletion versus a KANSL1 sequence variant. _European Journal of Human Genetics_, _24_(5), 652–659. https://doi.org/10.1038/ejhg.2015.178

Weidling, I., Preiss, C.N., Chancellor, S.E. _et al._ hiPSC-neurons recapitulate the subtype-specific cell intrinsic nature of susceptibility to neurodegenerative disease-relevant aggregation. _acta neuropathol commun_ **13**, 108 (2025). https://doi.org/10.1186/s40478-025-02000-4

Linda, K., Lewerissa, E. I., Verboven, A. H. A., Gabriele, M., Frega, M., Klein Gunnewiek, T. M., Devilee, L., Ulferts, E., Hommersom, M., Oudakker, A., Schoenmaker, C., van Bokhoven, H., Schubert, D., Testa, G., Koolen, D. A., de Vries, B. B. A., & Nadif Kasri, N. (2022). Imbalanced autophagy causes synaptic deficits in a human model for neurodevelopmental disorders. _Autophagy_, _18_(2), 423–442. https://doi.org/10.1080/15548627.2021.1936777

Frega, M., van Gestel, S. H., Linda, K., van der Raadt, J., Keller, J., Van Rhijn, J. R., Schubert, D., Albers, C. A., & Nadif Kasri, N. (2017). Rapid Neuronal Differentiation of Induced Pluripotent Stem Cells for Measuring Network Activity on Micro-electrode Arrays. _Journal of visualized experiments : JoVE_, (119), 54900. https://doi.org/10.3791/54900

Schuurmans, I. M. E., Mordelt, A., Linda, K., Puvogel, S., Duineveld, D., Hommersom, M. P., Rahm, L., Dyke, E., Scholten, G.-J., Knorz, C., Oudakker, A., Bokhoven, H. van, Lot, D.M, C., Garanto, A., & Nael Nadif Kasri. (2024). Navigating Human Astrocyte Differentiation: Direct and Rapid one-step Differentiation of Induced Pluripotent Stem Cells to Functional Astrocytes Supporting Neuronal Network development. _BioRxiv (Cold Spring Harbor Laboratory)_. https://doi.org/10.1101/2024.03.27.586938

Mordelt, A., Schuurmans, E., Scheefhals, N., Hommersom, M. P., Koen Slottje, Mast, K., Graziani, M., Gonzalez, C. O., Laura, Schubert, D., Nael Nadif Kasri, & Lot. (2025). Long-term co-maturation of stem cell-derived microglia and neuronal networks: an optimized platform to assess human microglial contribution to neuronal function. _BioRxiv (Cold Spring Harbor Laboratory)_. https://doi.org/10.1101/2025.10.24.684345